frontend/src/pages/AddQuestionPage/AddQuestion.tsx

```diff
@@ -53,34 +54,44 @@ function Add() {
   
     }
 
+
     return (
         <div className="homepage">
           <div className="background" />
     
           <div className="navbar">
-            <div className="title">Question Bank</div>
+            <div className="title">Peer Prep</div>
           </div>
     
-            <Form className="d-grid gap-2" style={{margin:"15rem", width: "80%", alignItems: "center"}}>
+            <Form className="d-grid gap-2" style={{margin:"8rem auto", width: "80%", alignItems: "center"}}>
 
                 <Form.Group className="mb-3" controlId="formName">
                     <Form.Control type="text" placeholder = "Enter Question Title" required onChange= {(e) => setTitle(e.target.value)}>
                     </Form.Control>
                 </Form.Group>
 
-                <Form.Group className="mb-3" controlId="formName">
-                    <Form.Control type="text" placeholder = "Enter Question Complexity" required onChange= {(e) => setComplexity(e.target.value)}>
+                <Form.Group className="mb-3" controlId="formComplexity">
+                    <Form.Control as="select" value={complexity} onChange={(e) => setComplexity(e.target.value)}>
+                    <option value="">Select Complexity</option>
+                    <option value="1">1</option>
+                    <option value="2">2</option>
+                    <option value="3">3</option>
                     </Form.Control>
```

Comment:
not indented properly for readability

![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g10/assets/72195240/8ec5c7a7-c677-4ac8-a4ff-bf9758086eee)

after auto-indent

![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g10/assets/72195240/983288fe-c6cb-4a2e-a202-80eaf6a35912)